Small update to instructional comments in app template

mit-mit opened the pull request:
- Add IntelliJ as an option under reload

Fixes: https://github.com/flutter/flutter/issues/7990

- Make a few comments fit one line

mit-mit committed:
Small update to instructional comments in app template

mit-mit commented:
@Hixie as discussed in #7990, I think this is subtle enough that it fits with our IDE being recommended.

mit-mit commented:
@sethladd please review

sethladd commented on packages/flutter_tools/templates/create/lib/main.dart.tmpl:
```diff
@@ -5,8 +5,7 @@ import 'package:flutter_driver/driver_extension.dart';
 {{#withDriverTest}}
-  // Starts the app with Flutter Driver extension enabled to allow Flutter Driver
-  // to test the app.
+  // Starts with Flutter Driver extension enabled allowing it to test the app.
```
Maybe: "Allow the Flutter Driver to test the app. See goo.gl/i83h2f for more info."

(I made up that link. Maybe point it to https://flutter.io/testing/ ?)

Hixie replied:
I don't really have a suggestion but I agree with Seth that the current text and the proposed text are both suboptimal here.

sethladd reviewed (commented):
other tweaks lgtm. Thanks!

Hixie commented:
I think this is fine, yeah. LGTM modulo seth's comment.

mit-mit committed:
Update Flutter Driver comment

Good suggestion to add the link, Seth! I also updated the working to match what that page uses.

mit-mit commented:
Driver comment updated, PTAL!

mit-mit commented:
@sethladd can you take a final look?

Hixie commented:
![LGTM](http://junkyard.damowmow.com/563)
